.changeset/slimy-actors-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': patch
+---
+
+Fixed thread message preview click behavior to navigate to the clicked reply message instead of unintentionally jumping to the parent message
```

Comment:
```suggestion
Fixes thread message preview click behavior, navigate to the clicked reply message instead of unintentionally jumping to the parent message
```